apis/io-engine/protobuf/v1/pool.proto

```diff
@@ -152,6 +173,14 @@ message PoolMetadataInfo {
   uint64 md_used_pages = 3;
 }
 
+// Pool RAID information.
+message RaidInfo {
+  // RAID level (e.g., "raid0", "raid1", "raid5").
```

Comment:
these could be enums?

Reply:
yes, it could be an enum. Just used a 'string' here so we don't have to modify the RPC when we want to introduce other raid types.